app/src/test/java/io/intrepid/contest/screens/contestresults/ContestResultsPresenterTest.java

```diff
@@ -0,0 +1,111 @@
+package io.intrepid.contest.screens.contestresults;
+
+import com.jakewharton.retrofit2.adapter.rxjava2.HttpException;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import io.intrepid.contest.models.RankedEntryResult;
+import io.intrepid.contest.rest.ContestResultResponse;
+import io.intrepid.contest.testutils.BasePresenterTest;
+import io.reactivex.Observable;
+
+import static io.reactivex.Observable.error;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class ContestResultsPresenterTest extends BasePresenterTest<ContestResultsPresenter> {
+    @Mock
+    ContestResultsContract.View mockView;
+
+    @Before
+    public void setup() {
+        presenter = new ContestResultsPresenter(mockView, testConfiguration);
+    }
+
+    @Test
+    public void onViewCreatedShouldSetupAdapter() {
+        setupUnsuccessfulContestResultsCall(); // Pre-requisite to handle mockView responses
```

Comment:
Doesn't matter if it's successful or not. It just matters that I return something in my mock methods, which is what I'm doing in `setupUnsuccessfulContestResultsCall()`. I'll just repeat that code for clarification then.